### ISSUE TITLE
Increased transferrin concentration

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6620]
**Preferred term label:** Increased transferrin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal transferrin concentration (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
These transferrin terms already exist, see https://hpo.jax.org/app/browse/term/HP:0032385
